client/go/outline/client_test.go

```diff
@@ -95,10 +95,10 @@ func Test_NewTransport_Multihop_URL(t *testing.T) {
 endpoint:
     $type: dial
     address: exit.example.com:4321
-    dialer: ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTpTRUNSRVQ@entry.example.com:4321/
+    dialer: ss://Y2hhY2hhMjAtaWV0Zi1wb2x5MTMwNTpTRUNSRVQ@example.com:4321/
```

Comment:
I'm not a fan of the tests depending on external infrastructure.
For now, can you revert the test and update the logic condition to add `!testing.Testing()`: https://pkg.go.dev/testing#Testing

We also need a TODO to clean this up.

Reply:
Done.